content/general/events/ifttt.textile

```diff
@@ -65,7 +65,7 @@ x-ably-envelope-source: channel.message
 x-ably-message-encoding: json
 x-ably-message-id: {UNIQUE_ABLY_MESSAGE_ID}
 x-ably-message-timestamp: {TIMESTAMP_ORIGINAL_MESSAGE_WAS_SENT}
-x-ably-version: 1.0
+x-ably-version: 1.2
```

Comment:
@SimonWoolf is this correct?  Will the Reactor rules use a 1.2 now?

Reply:
Not yet, as I mentioned in [my review of this pr](https://github.com/ably/docs/pull/843#pullrequestreview-384134783). I'm planning to change the default for new rules, per the new versioning policy you wrote. (I still think this change is unnecessary and that changing the version without there being any actual difference is confusing to customers, but I've already lost that argument 🙂)